docs/apps/lnw/es2k/es2k-lnw-overlay-ipv6.md

```diff
@@ -31,17 +31,19 @@ for more details on this feature.
 
 Prerequisites:
 
-- Download `hw-p4-programs` TAR file specific to the build and extract it to get `fxp-net_linux-networking-v2` p4 artifacts. Go through `Limitations` specified in `README` and bring up the setup accordingly.
-- Follow steps mentioned in [Deploying P4 Programs for E2100](/guides/es2k/deploying-p4-programs) for bringing up IPU with a custom P4 package.
+- For Linux Networking v3
+  - Download `hw-p4-programs` TAR file compliant with the CI build image and extract it to get `fxp-net_linux-networking-v3` P4 artifacts. Check readme for bring up guide, and be sure to check limitation for known issues.
+
+  - Follow steps mentioned in [Deploying P4 Programs for E2100](/guides/es2k/deploying-p4-programs) for bringing up IPU with a custom P4 package.
 Modify `load_custom_pkg.sh` with following parameters for linux_networking package:
 
-```text
-    sed -i 's/sem_num_pages = 1;/sem_num_pages = 25;/g' $CP_INIT_CFG
-    sed -i 's/lem_num_pages = 1;/lem_num_pages = 10;/g' $CP_INIT_CFG
-    sed -i 's/acc_apf = 4;/acc_apf = 8;/g' $CP_INIT_CFG
-```
+     ```text
+        sed -i 's/sem_num_pages = 1;/sem_num_pages = 28;/g' $CP_INIT_CFG
+        sed -i 's/lem_num_pages = 1;/lem_num_pages = 10;/g' $CP_INIT_CFG
+        sed -i 's/allow_change_mac_address = false;/allow_change_mac_address = true;/g' $CP_INIT_CFG
+        sed -i 's/acc_apf = 4;/acc_apf = 16;/g' $CP_INIT_CFG
 
-- Download `IPU_Documentation` TAR file specific to the build and refer to `Getting Started Guide` on how to install compatible `IDPF driver` on host. Once an IDPF driver is installed, bring up SRIOV VF by modifying the `sriov_numvfs` file present under one of the IDPF network devices. Example as below
+- Download `IPU_Documentation` TAR file compliant with the CI build image and refer to `Getting Started Guide` on how to install compatible `IDPF driver` on host. Once an IDPF driver is installed, bring up SRIOV VF by modifying the `sriov_numvfs` file present under one of the IDPF network devices. Example as below
```

Comment:
_compliant_ means "inclined to agree with others or obey rules."

_specific to_ is an appropriate expression. It conveys the idea that the instructions may change between versions of the image, so the reader needs to get the instructions for that image.

What's missing is a means for the reader to know which instructions go with which image. There are two ways to do this:

1. Package the image and its documentation together.
2. Provide a means (through version labels) to map the image to its documentation.